charts/charon-cluster/README.md.gotmpl

```diff
@@ -15,57 +15,35 @@ Charon Cluster
 
 {{ template "chart.valuesSection" . }}
 
-# How to use the chart
+# How to use this chart
 
-Installs [Charon](https://github.com/ObolNetwork/charon) full cluster
+A distributed validator cluster is a docker-compose file with the following containers running:
 
-## Add Obol's Helm Charts
+- Single execution layer client
+- Single consensus layer client
+- Number of Distributed Validator clients [This chart]
+- Number of Validator clients
+- Prometheus, Grafana and Jaeger clients for monitoring this cluster.
 
-```console
-helm repo add obol https://obolnetwork.github.io/helm-charts
-helm repo update
-```
-_See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+![Distributed Validator Cluster](https://github.com/ObolNetwork/charon-distributed-validator-cluster/blob/main/DVCluster.png?raw=true)
 
 ## Prerequisites
-- You completed the DKG ceremony and have generated the `.charon` directory.
-- The charon cluster keys are added to the Kubernetes cluster as secrets in the same namespace where the charon cluster is deployed.
+You have the followin charon artifacts created as k8s secrets per each charon node:
+- `<cluster-name>-<node-index>-validators` i.e `charon-cluster-0-validators`
+- `<cluster-name>-<node-index>-charon-enr-private-key` i.e `charon-cluster-0-charon-enr-private-key`
+The cluster lock is a single secret for the whole cluster:
+- `cluster-lock`
 
-### Example: How to create the k8s secrets from a `.charon` directory:
-```console
-cat << 'EOF' >> create-k8s-secrets.sh
-i=0
-CLUSTER_NAME=charon-cluster
-CHARON_CLUSTER_NAMESPACE=charon-cluster
-NODES=4
-# set current namespace
-kubectl config set-context --current --namespace=$CHARON_CLUSTER_NAMESPACE
-kubectl -n $CHARON_CLUSTER_NAMESPACE create secret generic cluster-lock --from-file=cluster-lock.json=.charon/cluster/cluster-lock.json --dry-run=client -o yaml | kubectl apply -f -
-while [[ $i -lt "$NODES" ]]
-do
-    files=""
-    for secret in .charon/cluster/node${i}/validator_keys/*; do
-        files="$files --from-file=.charon/cluster/node${i}/validator_keys/$(basename $secret)"
-    done
-    kubectl -n $CHARON_CLUSTER_NAMESPACE create secret generic ${CLUSTER_NAME}-${i}-validators $files --dry-run=client -o yaml | kubectl apply -f -
-    kubectl -n $CHARON_CLUSTER_NAMESPACE create secret generic ${CLUSTER_NAME}-${i}-charon-enr-private-key --from-file=charon-enr-private-key=.charon/cluster/node${i}/charon-enr-private-key --dry-run=client -o yaml | kubectl apply -f -
-    ((i=i+1))
-done
-EOF
-chmod +x create-k8s-secrets.sh && ./create-k8s-secrets.sh
-```
```

Comment:
They didn't think it was useful to give an example?

Reply:
yeah, Oisin believes it is a complex bash script to give as an example, and it is better if we do not give a direction on how to create secrets.
Also in the next feedback cycle, we should figure out a solution to auto-generate and consume the cluster secret as part of the chart itself, not with an external bash script.